themes/inpycon2020/templates/base.html

```diff
@@ -10,6 +10,19 @@
     <link rel="shortcut icon" href="/blog/theme/images/favicon.png"/>
     <link rel="stylesheet" href="/blog/theme/css/bootstrap.min.css"/>
     <link rel="stylesheet" href="/blog/theme/css/styles.css"/>
+    {% if article is defined %}
+    <meta property="og:type" content="article" />
+    <meta property="og:url" content="{{ SITEURL }}/{{ article.url }}" />
+    <meta property="og:title" content="{{ article.title | replace("\"", "&quot;") }}" />
+    <meta property="og:description" content="{{ article.content | striptags | replace("\"", "&quot;") | truncate(124, False, '...') }}" />
+    <meta property="og:image" content="https://in.pycon.org/blog/theme/images/opengraph-default.png" />
```

Comment:
Use `{{ SITEURL }}` here plus the resource (theme/images/opengraph-default.png) doesn't exist as well as not added in this PR.

Am I missing something here? @vipulgupta2048